frontend/app/src/components/history/ExternalTradeForm.vue

```diff
@@ -377,29 +398,31 @@ onMounted(() => {
                   {{ baseHint }}
                 </div>
                 <asset-select
-                  v-model="base"
+                  v-model="baseAsset"
                   outlined
                   required
                   data-cy="base-asset"
-                  :error-messages="v$.base.$errors.map(e => e.$message)"
+                  :error-messages="getErrorMessages(v$.baseAsset)"
                   :hint="t('external_trade_form.base_asset.hint')"
                   :label="t('external_trade_form.base_asset.label')"
-                  @focus="delete errorMessages['baseAsset']"
+                  @focus="removeError('baseAsset')"
+                  @blur="v$.baseAsset.$touch()"
                 />
               </v-col>
               <v-col cols="12" md="6" class="d-flex flex-row align-center">
                 <div class="text--secondary external-trade-form__action-hint">
                   {{ quoteHint }}
                 </div>
                 <asset-select
-                  v-model="quote"
+                  v-model="quoteAsset"
                   required
                   outlined
                   data-cy="quote-asset"
-                  :error-messages="v$.quote.$errors.map(e => e.$message)"
+                  :error-messages="getErrorMessages(v$.quoteAsset)"
                   :hint="t('external_trade_form.quote_asset.hint')"
                   :label="t('external_trade_form.quote_asset.label')"
-                  @focus="delete errorMessages['quoteAsset']"
+                  @focus="removeError('quoteAsset')"
```

Comment:
What happens if we don't explicitly do this?

Reply:
If we use the old way (delete errorMessage), it doesn't change the $error variable. I think it because it doesn't trigger the reactivity.

Reply:
ok let me think about it a bit, it's because this has to update the property outside.